test/index.cpp

```diff
@@ -38,6 +38,58 @@ int main()
             CHECK(ranges::equal(ranges::view::c_str(e.what()),
                                 ranges::view::c_str("ranges::at")));
         }
+
+        try {
+            ranges::at(vi, -1);
+            CHECK(false);
+        } catch(std::out_of_range const& e) {
+            CHECK(ranges::equal(ranges::view::c_str(e.what()),
+                                ranges::view::c_str("ranges::at")));
+        }
+
+        auto viv = ranges::make_iterator_range(vi.begin(), vi.end());
+        CHECK(viv.at(0) == 1);
+        CHECK(viv.at(1) == 2);
+        CHECK(viv.at(2) == 3);
+        CHECK(viv.at(3) == 4);
+        
+        try {
+            viv.at(4);
+            CHECK(false);
+        } catch(std::out_of_range const& e) {
+            CHECK(ranges::equal(ranges::view::c_str(e.what()),
+                                ranges::view::c_str("view_interface::at")));
+        }
+
+        try {
+            viv.at(-1);
+            CHECK(false);
+        } catch(std::out_of_range const& e) {
+            CHECK(ranges::equal(ranges::view::c_str(e.what()),
+                                ranges::view::c_str("view_interface::at")));
+        }
+
+        const auto cviv = viv;
+        CHECK(cviv.at(0) == 1);
+        CHECK(cviv.at(1) == 2);
+        CHECK(cviv.at(2) == 3);
+        CHECK(cviv.at(3) == 4);
+        
+        try {
+            cviv.at(4);
+            CHECK(false);
+        } catch(std::out_of_range const& e) {
+            CHECK(ranges::equal(ranges::view::c_str(e.what()),
+                                ranges::view::c_str("view_interface::at")));
+        }
+
+        try {
+            cviv.at(-1);
+            CHECK(false);
+        } catch(std::out_of_range const& e) {
+            CHECK(ranges::equal(ranges::view::c_str(e.what()),
+                                ranges::view::c_str("view_interface::at")));
+        }
```

Comment:
From Wikipedia:
> This style puts the brace associated with a control statement on the next line, indented to the same level as the control statement.

So this:

```c++
try
{
    ...
}
catch(...)
{
    ...
}
```

Just update your own tests. If there are other tests that are ill-formatted, leave them and we can update them in a separate PR.

Thanks.